.github/workflows/dep_fuzzing.yml

```diff
@@ -60,3 +61,9 @@ jobs:
         with:
           name: fuzz-crash-artifacts
           path: fuzz/artifacts/
+
```

Comment:
This should be in the file `.github/workflows/Fuzzing.yml`, we don't want to to create an issue if it fails in a PR run only on the cron job